Licensing_Program/userfiles_handling.py

```diff
@@ -177,3 +179,16 @@ def commentfmt_userfile(userfile_path, config):
         ),
         None
     )
+
+
+def shebang_check(user_file_name, line):
+    """If the variable line begins with the character '#!', an exception is
+    raised.
+    """
+    if line.startswith('#!'):
+        raise ValueError(
+            "Header was specified to be inserted at line in file {}. "
```

Comment:
Change _`at line in`_ to _`at line 1 in`_.

Although, I'm not sure this needs to be a function. We aren't going to use it anywhere else, and most of the lines are the two strings, one of which would be removed if inlined.

And what do you think of changing the error message to:

    "File {} starts with a shebang (#!). To insert the header before it, use `--ignore-shebang`.
    Otherwise, change the `InsertAtLine` config value."